src/js/configs.js

```diff
@@ -179,17 +176,14 @@ module.exports = {
     highlighter: 'ini',
     latestVersion: '5.71',
     name: 'stunnel',
-    showSupports: false,
     supportsHsts: false,
     supportsOcspStapling: false,
     tls13: '5.50',
-    usesOpenssl: true,
   },
-tomcat: {
+  tomcat: {
     highlighter: 'xml',
-    latestVersion: '9.0.30',
+    latestVersion: '9.0.96',
```

Comment:
Keeping this on v9 branch as that is basically the mainline, with v10.1.x IIRC only changing its licensing(?), v10.0.x EOL and v11 not yet stable (and, unknown config compatibility FWIW)